core/src/query.rs

```diff
@@ -160,9 +166,17 @@ impl MongoQuery {
 
             let mut schema_catalog_doc = schema_catalog_doc_vec[0].to_owned();
 
-            let collections_schema_doc = schema_catalog_doc
-                .get_document_mut(current_db)
-                .map_err(|e: ValueAccessError| Error::ValueAccess(current_db.to_string(), e))?;
+            let collections_schema_doc =
+                schema_catalog_doc
+                    .get_document_mut(current_db)
+                    .map_err(|e| {
+                        log::warn!(
+                            "Failed to get schema document for database `{0}`: {1}",
+                            current_db,
+                            e
+                        );
+                        Error::SchemaValueAccess(current_db.to_string(), e)
```

Comment:
In looking at the errors, I determined this wasn't the same as a ValueAccess error elsewhere, since it his highly localized to schema reads.